index.html

```diff
@@ -115,41 +115,62 @@ <h1>More About HackYU</h1>
             <p class="rules">
                 Rules:
             </p>
+
             <ol class="rules">
                 <li> Maximum of 5 people per team</li>
                 <li> Only one project submission per team</li>
             </ol>
         </div>
     </div>
-
+    <h1>Workshop Instructors</h1>
     <div id="guests">
```

Comment:
```suggestion
    <br>
    <div id="guests">
```
It might be good to add a small space between the pictures and the section name